Doc/library/multiprocessing.rst

```diff
@@ -1245,7 +1245,6 @@ Connection objects are usually created using
       Connection objects themselves can now be transferred between processes
       using :meth:`Connection.send` and :meth:`Connection.recv`.
 
-   .. versionadded:: 3.3
       Connection objects now support the context management protocol -- see
```

Comment:
Since this is a distinct change from the previous paragraph:

```suggestion
      Connection objects also now support the context management protocol -- see
```

Reply:
Thanks for the suggestion.